[
{
	"id":"Babel #1134", "type":"Logical", "url":"https://github.com/python-babel/babel/issues/1134", "lead":"Gabe Sherman", "date":"2024-09-24", "desc":"List index out of range error in PoFileParser._add_message"
},
{
	"id":"CPython #125648", "type":"Logical", "url":"https://github.com/python/cpython/issues/125648", "lead":"Gabe Sherman", "date":"2024-10-17", "desc":"TypeError in email.message.Message.get_params"
},
{
	"id":"Arrow-Py #1191", "type":"Logical", "url":"https://github.com/arrow-py/arrow/issues/1191", "lead":"Gabe Sherman", "date":"2024-10-04", "desc":"IndexError in DateTimeParser.parse"
},
{
	"id":"Pillow #8597", "type":"Logical", "url":"https://github.com/python-pillow/Pillow/issues/8597", "lead":"Gabe Sherman", "date":"2024-12-13", "desc":"ZeroDivisionError occurs when calling Image.open"
},
{
	"id":"DateUtil #1402", "type":"Logical", "url":"https://github.com/dateutil/dateutil/issues/1402", "lead":"Gabe Sherman", "date":"2024-12-10", "desc":"TypeError: unsupported operand type(s) for *=: 'NoneType' and 'int' when calling tz.gettz"
}
]